keel-retrofit/src/main/kotlin/com/netflix/spinnaker/keel/retrofit/KeelRetrofitConfiguration.kt

```diff
@@ -69,36 +69,11 @@ class KeelRetrofitConfiguration {
       ))
       retryOnConnectionFailure(okHttpClientProperties.isRetryOnConnectionFailure)
     }
-    return cfg.build()
-  }
+    .build()
 
   @Bean
   fun retrofitLoggingInterceptor(@Value("\${retrofit2.log-level:BASIC}") retrofitLogLevel: String) =
     HttpLoggingInterceptor().apply {
       level = HttpLoggingInterceptor.Level.valueOf(retrofitLogLevel)
     }
-
-  // TODO: this should only be applied to spinnaker-to-spinnaker requests
-  @Bean
-  fun spinnakerRequestInterceptor(
-    okHttpClientConfigurationProperties: OkHttpClientConfigurationProperties
-  ) =
-    // Inline version of SpinnakerRequestInterceptor from kork-web
-    Interceptor { chain ->
-      val requestBuilder = chain.request().newBuilder()
-      if (okHttpClientConfigurationProperties.propagateSpinnakerHeaders) {
-        AuthenticatedRequest
-          .getAuthenticationHeaders()
-          .forEach { (k, v) ->
-            v.ifPresent {
-              requestBuilder.addHeader(k, it)
-            }
-          }
-      }
-      try {
-        AuthenticatedRequest.allowAnonymous { chain.proceed(requestBuilder.build()) }
```

Comment:
We shouldn't need to do this as we authenticate all requests using the Keel service account.